docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -626,4 +626,12 @@ The issue "SyntaxError: 'yield' inside list comprehension" might occur during co
 The following workarounds are suggested to resolve this issue:
 1. Use Python 3.6/3.7 to convert MXNet\* models on Windows
 2. Update MXNet: pip install mxnet=1.7.0.post2
-Note that you might have conflicts between previously installed PyPI dependencies.
\ No newline at end of file
+Note that you might have conflicts between previously installed PyPI dependencies.
+
+#### 105. What does the message "The IR preparation was executed by the legacy MO path. ..." mean? <a name="question-105"></a>
+
+For some model formats (in current state for ONNX and TensorFlow) are available two paths of IR conversion. 
+The old one is handled by old python implementation while the new one is realized via new c++ frontends. 
+Since version 2022.1 the default path of IR conversion for ONNX models is handled via new ONNX frontend. 
+Some features like `--extensions` and `--transformations_config` are not supported yet on the new frontends. 
+If a user doesn't choose expected path of conversion explicitly (by `--use_new_frontend` or `--use_legacy_frontend` MO arguments) and some not supported pre-defined scenario is detected on the new frontend path, the IR conversion falls back to the old path.
```

Comment:
The IR conversion is performed via the old path in the following cases: 
- a user does not select the new expected path of conversion explicitly by `--use_new_frontend` or `--use_legacy_frontend` MO arguments;
- an unsupported pre-defined scenario is detected on the new frontend path.

Reply:
It rather should be:
The IR conversion is performed via the old path if a user does not select ~the new~ *any* expected path of conversion explicitly (by `--use_new_frontend` or `--use_legacy_frontend` MO arguments) and an unsupported pre-defined scenario is detected on the new frontend path.
I am not sure if the word *fallback* shouldn't be used here explicitly (it is rather common in developer slogan, but I can be wrong)

Reply:
Ok then, agreed, let's say:
The IR conversion falls back to the old path if a user does not select any expected path of conversion explicitly (by --use_new_frontend or --use_legacy_frontend MO arguments) and unsupported pre-defined scenario is detected on the new frontend path.